filebeat/docs/migration.asciidoc

```diff
@@ -200,20 +199,20 @@ output:
     # The Logstash hosts. <1>
     hosts:
       - localhost:5043
-      
+
     # Network timeout in seconds.
     timeout: 15
 
     tls: <2>
       # List of root certificates for HTTPS server verifications
-      certificate_authorities: 
+      certificate_authorities:
         - ./logstash-forwarder.crt
-        
+
       # Certificate for TLS client authentication
       certificate: ./logstash-forwarder.crt
 
       # Client Certificate Key
-      certificate_key: ./logstash-forwarder.key    
+      certificate_key: ./logstash-forwarder.key
```

Comment:
btw. for SSL we don't need certificate and certificate_key. These two shall be used for client authentication.